package.json

```diff
@@ -73,6 +73,7 @@
     "react-router-dom": "5.3.3",
     "react-select": "2.4.4",
     "react-transition-group": "4.4.2",
+    "react-use": "17.4.0",
```

Comment:
What were we planning to use this package for?

Reply:
That was supposed to be removed.